datamodel_code_generator/parser/jsonschema.py

```diff
@@ -367,6 +367,7 @@ def _get_type(type_: str, format__: Optional[str] = None) -> Types:
 @snooper_to_methods(max_variable_length=None)
 class JsonSchemaParser(Parser):
     SCHEMA_PATHS: ClassVar[List[str]] = ['#/definitions', '#/$defs']
+    SCHEMA_OBJECT_TYPE: ClassVar[Type[JsonSchemaObject]] = JsonSchemaObject
```

Comment:
@WilliamJamieson 
Why not just pass it through init and make it an instance variable?
This is not a criticism but a simple question.

Reply:
There is no particular reason. My application already required me to make a subclass of the parser itself to introduce some other behaviors that I need. So it makes no difference to me whether it's passed in through the `__init__` or a class variable.

I figured this type of change could drastically change the parser behavior. So forcing users to do it via a subclass made sense. L

Reply:
make sense!!